styles/styles.css

```diff
@@ -0,0 +1,389 @@
+* {box-sizing: border-box;}
+
+* {font-family: Georgia, serif;}
+
+
+header {
+  display: flex;
+
+  flex-direction: column;
+  padding: 2em;
+
+  background: rgb(115,136,153);
+  background: linear-gradient(0deg, rgba(115,136,153,1) 0%, rgba(9,60,102,1) 100%);
+  background-size: auto;
+  background-image: url('../images/header-image.jpg');
+
+
+
+}
+
+
+.logo {
+  width: 70%;
+}
+
+.logo img {
+  width: 60%;
+}
+
+.logo h1 {
```

Comment:
Awesome! Bold use of em's for this course. It's kind of an advanced way of doing things. Hopefully it wasn't too hard to work with.